### PR TITLE
[BDRSPS-465] Updated `schema.json` for `survey_metadata`; unit tests updated

### PR DESCRIPTION
**Pull request by chungvl**

### CODE REVIEW

File: abis_mapping/templates/survey_metadata/schema.json [hunk 1]
```diff
@@ -56,8 +66,8 @@
       }
     },
     {
-      "name": "taxonomicCoverage",
-      "title": "Taxonomic Coverage",
+      "name": "targetTaxonomicScope",
+      "title": "Target Taxonomic Scope",
       "description": "The range of biological taxa covered by the survey. Multiple terms are allowed, separated by a vertical bar aka pipe |",
       "example": "Coleoptera | Insecta",
       "type": "string",
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_metadata/schema.json:73]: This should be `list` type.

File: abis_mapping/templates/survey_metadata/schema.json [hunk 2]
```diff
@@ -66,6 +76,17 @@
         "required": false
       }
     },
+    {
+      "name": "targetHabitatScope",
+      "title": "Target Habitat Scope",
+      "description": "The habitats targeted for sampling during the survey. Multiple terms are allowed, separated by a vertical bar aka pipe |",
+      "example": "Woodland | Tundra",
+      "type": "string",
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_metadata/schema.json:84]: should be `list`

File: abis_mapping/templates/survey_metadata/schema.json [hunk 3]
```diff
@@ -101,9 +122,31 @@
...
     },
     {
-      "name": "surveyMethodBibliographicReferences",
-      "title": "Survey Method Bibliographic References",
-      "description": "Bibliographic references of survey methods used.",
+      "name": "surveyOrganisation",
+      "title": "Survey Organisation",
+      "description": "Name of organisation or individual for whom Survey is being conducted. Multiple terms are allowed, separated by a vertical bar aka pipe |",
+      "example": "NSW Department of Planning, Industry and Environment | CSIRO",
+      "type": "string",
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_metadata/schema.json:129]: should be `list`

File: abis_mapping/templates/survey_metadata/schema.json [hunk 4]
```diff
@@ -101,9 +122,31 @@
...
+      "constraints": {
+        "required": false
+      }
+    },
+    {
+      "name": "samplingPerformedBy",
+      "title": "Sampling Performed By",
+      "description": "Similar to eco:samplingPerformedBy. Individual/s that carried out the survey. These can be represented by identifiers. Multiple values allare allowed, separated by a vertical bar aka pipe |",
+      "example": "https://orcid.org/0000-0003-1092-9705 | https://orcid.org/0009-0004-5278-4238",
+      "type": "string",
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_metadata/schema.json:140]: should be `list`

File: abis_mapping/templates/survey_metadata/mapping.py [hunk 5]
```diff
@@ -75,7 +75,13 @@ def apply_validation(
...
-                            "spatialCoverageGeodeticDatum",
+                            "geodeticDatum",
+                        ]
+                    ),
+                    plugins.mutual_inclusion.MutuallyInclusive(
+                        field_names=[
+                            "samplingEffortValue",
+                            "samplingEffortUnit",
                         ]
                     ),
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_metadata/mapping.py:86]: :+1:

**Comment by joecrowleygaia** [comment]
There is a need to review the mapping for the fields that have now become `lists`, e.g. should they be iterated and turned into separate nodes when graphing, or reformed back into a string as a JSON array (which is what is happening now), or back to a '|' separated string. Mapping tests now failing for metadata because of this.

File: abis_mapping/templates/survey_metadata/mapping.py [hunk 6]
```diff
@@ -273,18 +279,35 @@ def add_survey(
...
         # Add keywords
         if keywords := row["keywords"]:
             for keyword in keywords:
                 graph.add((uri, rdflib.SDO.keywords, rdflib.Literal(keyword)))
 
+        # Add Orgs
+        if organisations := row["surveyOrganisation"]:
+            for organisation in organisations:
+                graph.add((uri, utils.namespaces.BDR.organization, rdflib.Literal(organisation)))
+
```
Comment (chungvl) [inline abis_mapping/templates/survey_metadata/mapping.py:304]: @joecrowleygaia  what do you think this (above and below)?

Comment (joecrowleygaia) [reply abis_mapping/templates/survey_metadata/mapping.py:304]: these are all new mappings? it's probably the way to go and i'd love to approve for completeness but there are the abis data standards that need to be adhered to which mieke is the main contact for us on such things. I'm happy with all the changes that happened yesterday so if you want you can revert this latest commit and i can merge. Then i'd recommend raising with Mieke at stand up tomorrow or via a backlog ticket at least to have these fields that haven't been mapped, to be.

Comment (chungvl) [reply abis_mapping/templates/survey_metadata/mapping.py:304]: Yep, they were the new mappings that were not already covered by the previous branch.  I'll roll it back and save it for later :)

**Review by joecrowleygaia — APPROVED** [review]

**Pull request merged**